dockerproxy/main.go

```diff
@@ -470,6 +472,35 @@ func authorizeRequest(appName, authToken string) bool {
 func extendDeadline() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		log.Infof("extendDeadline called with user agent: %s", r.UserAgent())
+
+		di, err := disk.GetInfo("/data")
```

Comment:
Is this "slow"? If extend deadline is too slow, I expect it might race w/ the deadline finishing before it is extended.

Reply:
No. It calls statfs(). So is more like `df` than `du`.

https://github.com/minio/minio/blob/ce3d9dc9faa5163d46f672fe2dbd8cb03e5056e5/pkg/disk/stat_linux.go#L30